spec/src/main/asciidoc/javase/bootstrap_se.asciidoc

```diff
@@ -1,61 +1,133 @@
-[[bootstrap-se]]
+[[se_bootstrap]]
 
 == Bootstrapping a CDI container in Java SE
 
-In Java SE, the CDI container must be explicitly bootstrapped by the application.
-This is performed with `CDIProvider` and `CDI` utility classes.
+In Java SE, the CDI container must be explicitly bootstrapped by the user.
+This is performed with the `SeContainerInitializer` abstract class and its static method `newInstance()`.
 
+`SeContainerInitializer` implementation is obtained by java service-provider.
+It allows the configuration of the CDI container before its bootstrapping.
+Calling the `initialize()` method bootstraps the container and return a `SeContainer`.
```

Comment:
The `SeContainerInitializer.initialize()` method bootstraps the container and returns a `SeContainer` instance.